docs/adr/0016-pr-review.md

```diff
@@ -0,0 +1,64 @@
+# Reviewing Pull Requests
+
+* Status: accepted
+* Deciders: tumido, HumairAK, lars, goern, 4n4nd
+* Date: 2021-05-20
+
+Technical Story: [#93](https://github.com/operate-first/blueprint/issues/93)
+
+## Context and Problem Statement
+
+Each community has its own `contributing.md` file which outlines how are users expected to contribute to the community. That guide for contributions should also clearly state what will happen once a PR is submitted and how reviews are performed.
+
+
+## Considered Options
+
+* Using inherited processes from AI-CoE - Prow
+* Building upon Prow and modifying and evolving them to operations needs
+
+## Decision Outcome
+
+Chosen option: "Building upon Prow and modifying and evolving them to operations needs"
+
+### Prerequisistes
+
+We use Prow to automate review process. That means we use following setup:
+
+1. Each repository contains an `OWNERS` file:
+   1. There is always one `OWNERS` file in a repository root folder
+   2. If needed additional `OWNERS` files can be created within nested folders. This file takes precedence for all resources within this folder.
+2. `OWNERS` file contains several entries:
+   1. `approvers` is a list of GitHub users that are allowed to use `/approve` chat-ops command. Role of approvers is to gate PRs - they verify that changes in PRs respect our architectural decisions and are in line with the heading of the project.
+   2. `reviewers` is a list of GitHub users that are allowed to use `/lgtm` chat-ops command. Their role is to ensure changes in PRs are implemented properly and meet our coding standards.
+
+### Process
+
+1. A PR is submited
+2. Prow (represented by a Sesheta bot-account) will pick a selection of reviewers from corresponding `OWNERS` file (based on modified files in the PR).
```

Comment:
Is this how it's currently done? or is this how we want it done? the reviewer selection has seemed a bit random to me, I was under the impression it was just doing a round robin

Reply:
What I meant with this is that the "selection" is based on `reviewers` list from `OWNERS` file. And which `OWNERS` file  is used is determined based on the modified files. Which `reviewers` are selected is pretty random.

Reply:
I thought it's round robin, but it can be "smarter" if  there's multiple owners files

Reply:
Ahh gotcha